### PR TITLE
protocol integration guide review

### PR DESCRIPTION
**Pull request by JoshuaAdemola**

### CODE REVIEW

File: content/protocol-integration/lora-loriot-bundle/loriot-troubleshooting.md [hunk 1]
```diff
@@ -16,4 +16,4 @@ Make sure that the **Gateway Information** is enabled in LORIOT Network Server s
 #### Device type error warning
 
 The warning message "Device type error" shows up in the log if no device protocol has been assigned to the device.
-To assign device protocol please refer the section [Set device protocol](#assign-loriot-device-protocol)
+To assign device protocol please refer the section [Assign the Loriot LoRa device protocol](#assign-loriot-device-protocol).
```
Comment (BeateRixen) [inline content/protocol-integration/lora-loriot-bundle/loriot-troubleshooting.md:19]: To assign device protocol please refer the section > To assign a device protocol refer to the section

File: content/protocol-integration/lwm2m-bundle/register-device.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ title: Registering LWM2M devices
 layout: redirect
 ---
 
-To register a LWM2M device in {{< product-c8y-iot >}}, upload a CSV file with registration data in the bulk registration dialog in Devices > Registration > Register device > Bulk device registration in the Device Management application
+To register a LWM2M device in {{< product-c8y-iot >}}, upload a CSV file with registration data in the bulk registration dialog in **Devices** > **Registration** > **Register device** > **Bulk device registration** in the Device Management application
```
Comment (BeateRixen) [inline content/protocol-integration/lwm2m-bundle/register-device.md:7]: Device Management application > Device Management application,

File: content/protocol-integration/opcua-bundle/device-protocols.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ When using a regular expression, keep in mind that it might be matching many nod
...
+> **Info:** If no reference server was selected during the device protocol creation, the path should be given with a namespace URI representation. In the OPC UA server the index value can be taken from the namespace array. An example namespace URI representation for browse path "5:Counter1" would be: `http://www.prosysopc.com/OPCUA/SimulationNodes:Counter1`. Node id equal to "ns=5;s=Simulation" will have the following namespace representation *'nsu=http://www.prosysopc.com/OPCUA/SimulationNodes;s=Simulation*. In both examples the server's namespace array, the 5th element has the value of `http://www.prosysopc.com/OPCUA/SimulationNodes`.
```
Comment (BeateRixen) [inline content/protocol-integration/opcua-bundle/device-protocols.md:33]: Not sure on this one. What is your take @rssag ? We usually use italics for URL but perhaps it makes sense to use code here instead?

File: content/protocol-integration/opcua-bundle/operations.md [hunk 4]
```diff
@@ -719,8 +719,6 @@ This returns a JSON response like this:
 
 Now download is possible with the self link provided inside the managedObjects section of the JSON response.
 
-For further information, refer to [Binaries > Binaries collection](/reference/binaries/#binaries-collection) in the *Reference guide*.
-
 For 10.9 and later, refer to [binaries API](https://www.{{< domain-c8y >}}/api/{{< c8y-current-version >}}/#tag/Binaries) in the {{< openapi >}}.
```
Comment (BeateRixen) [inline content/protocol-integration/opcua-bundle/operations.md:722]: For 10.9 and later, refer > For further information, refer to

File: content/protocol-integration/sigfox-bundle/register-device-sigfox.md [hunk 5]
```diff
@@ -21,7 +21,7 @@ In the next window, fill in the required information:
 - **PAC:** Porting authorization code for your device. The value must be a hexadecimal number.
 - **Contract:** Choose your desired contract (contracts that are active and with free slots are listed).
 - **Device protocol:** Select your desired device protocol from the drop-down list.
-- **Product certificate key:** This key can be located in *https://partners.sigfox.com/*. Navigate to your device and copy the certificate key. If the checkbox is not selected and no product certificate key is specified, the device will be considered a prototype.
+- **Product certificate key:** This key can be located in https://partners.sigfox.com/. Navigate to your device and copy the certificate key. If the checkbox is not selected and no product certificate key is specified, the device will be considered a prototype.
```
Comment (BeateRixen) [inline content/protocol-integration/sigfox-bundle/register-device-sigfox.md:24]: We actually use italics notation for URLs

File: content/protocol-integration/lora-loriot-bundle/loriot-troubleshooting.md (hunk 1 above)
Comment (JoshuaAdemola) [reply content/protocol-integration/lora-loriot-bundle/loriot-troubleshooting.md:19]: Fixed

File: content/protocol-integration/opcua-bundle/operations.md (hunk 4 above)
Comment (JoshuaAdemola) [reply content/protocol-integration/opcua-bundle/operations.md:722]: Fixed

File: content/protocol-integration/sigfox-bundle/register-device-sigfox.md (hunk 5 above)
Comment (JoshuaAdemola) [reply content/protocol-integration/sigfox-bundle/register-device-sigfox.md:24]: Fixed

File: content/protocol-integration/opcua-bundle/device-protocols.md (hunk 3 above)
Comment (JoshuaAdemola) [reply content/protocol-integration/opcua-bundle/device-protocols.md:33]: before

after

**Review by MWindel — APPROVED** [review]

File: content/protocol-integration/opcua-bundle/device-protocols.md (hunk 3 above)
Comment (JoshuaAdemola) [reply content/protocol-integration/opcua-bundle/device-protocols.md:33]: Fixed

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**